http/src/main/resources/init-resources/startup.sh

```diff
@@ -263,5 +315,10 @@ if [ ! -z "$CRYPTO_DETECTOR_DOCKER_IMAGE" ] ; then
 fi
 
 # Resize persistent disk if needed.
-echo "Resizing persistent disk attached to runtime $GOOGLE_PROJECT / $CLUSTER_NAME if disk size changed..."
-resize2fs /dev/${DISK_DEVICE_ID}
+# This condition assumes Dataproc's cert directory is different from GCE's cert directory, a better condition would be
+# a dedicated flag that distinguishes gce and dataproc. But this will do for now
+# If it's GCE, we resize the PD. Dataproc doesn't have PD
+if [ -f "$FILE" ]; then
```

Comment:
maybe make a ticket to template the cloudService in here, would be nice not too rely on this as it could change and break this in an unexpected way;

Reply:
https://broadworkbench.atlassian.net/browse/IA-3186